model/src/main/kotlin/ScanSummary.kt

```diff
@@ -60,21 +60,24 @@ data class ScanSummary(
     /**
```

Comment:
While I am ok with your change, please note that @sschuberth raised the same concerns for consistency. However I though it could be skipped here because few scanners need snippet findings:
https://github.com/oss-review-toolkit/ort/pull/6791#discussion_r1191042314